d2l-button-shared-styles.html

```diff
@@ -1,7 +1,14 @@
 <link rel="import" href="../polymer/polymer.html">
 <style is="custom-style">
 	html {
+		--d2l-button-clear-focus: {
+			box-shadow: 0 0 0 4px rgba(0, 0, 0, 0);
+		};
+		--d2l-button-focus: {
+			box-shadow: 0 0 0 4px var(--d2l-color-celestine-light);
+		};
 		--d2l-button-spacing: 0.75rem;
+		--d2l-color-celestine-light: rgba(0, 111, 191, 0.3);
```

Comment:
Does `--d2l-button-clear-focus` need to be a mixin or could it just be inlined below? Same for `--d2l-color-celestine-light` -- if it can be inlined inside the `--d2l-button-focus` mixin that'll speed things up. Mixins and variables need to be polyfilled by Polymer, so they're not currently free.

For that matter, I might even duplicate the 1 line of CSS in `--d2l-button-focus` so that it doesn't have to be a mixin and copy it into the 2 places that need it.